content/bn/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+
+* এজাইল অ্যাপ্লিকেশন তৈরি এবং স্থাপনা: ভিএম ইমেজ ব্যবহারের তুলনায় কন্টেইনার ইমেজ তৈরির সহজতা এবং দক্ষতা বেশি। 
+* ক্রমাগত বিকাশ, একীকরণ এবং স্থাপনা: দ্রুত এবং দক্ষ রোলব্যাকগুলির সাথে নির্ভরযোগ্য এবং ঘন ঘন কন্টেইনার ইমেজ তৈরি এবং স্থাপনার ব্যবস্থা করে (চিত্র অপরিবর্তনীয়তার কারণে).
+* ডেভ এবং অপস উদ্বেগের বিচ্ছেদ: স্থাপনার সময় না করে বিল্ড/রিলিজ সময়ে অ্যাপ্লিকেশন কন্টেইনার ইমেজ তৈরি করুন, যার ফলে অবকাঠামো থেকে অ্যাপ্লিকেশনগুলিকে বিচ্ছেদ করা যায়। 
+* পর্যবেক্ষণযোগ্যতা: শুধুমাত্র OS-স্তরের তথ্য এবং মেট্রিক্সই নয়, প্রয়োগের স্বাস্থ্য এবং অন্যান্য সংকেতও।
+* ডেভেলপমেন্ট, টেস্টিং এবং প্রোডাকশন জুড়ে পরিবেশগত সামঞ্জস্য: একটি ল্যাপটপে ক্লাউডের মতোই চলে।
+* ক্লাউড এবং ওএস ডিস্ট্রিবিউশন পোর্টেবিলিটি: Ubuntu, RHEL, CoreOS, on-premises, on major public clouds, এবং অন্য কোথাও চলে।
+* অ্যাপ্লিকেশন-কেন্দ্রিক ব্যবস্থাপনা: ভার্চুয়াল হার্ডওয়্যারে একটি OS চালানো থেকে লজিক্যাল রিসোর্স ব্যবহার করে একটি OS-এ একটি অ্যাপ্লিকেশন চালানো পর্যন্ত বিমূর্ততার স্তর বাড়ায়।
+* ঢিলেঢালাভাবে সংযুক্ত, বিতরণ করা, স্থিতিস্থাপক, মুক্ত মাইক্রো-পরিষেবা: অ্যাপ্লিকেশনগুলিকে ছোট, স্বাধীন টুকরোগুলিতে বিভক্ত করা হয় এবং গতিশীলভাবে স্থাপন ও পরিচালনা করা যায় – একটি বড় একক-উদ্দেশ্য মেশিনে চলমান একটি মনোলিথিক স্ট্যাক নয়।.
+* সম্পদ বিচ্ছিন্নতা: অনুমানযোগ্য অ্যাপ্লিকেশন কর্মক্ষমতা।
+* সম্পদের ব্যবহার: উচ্চ দক্ষতা এবং ঘনত্ব।
+
+## আপনার কেন কুবারনেটিস দরকার এবং এটি কী করতে পারে {#কেন-আপনার-দরকার-কুবারনেটিস-এবং-কী-পারে-এটি-করতে}
+
+কন্টেইনারগুলি আপনার অ্যাপ্লিকেশনগুলিকে বান্ডিল এবং চালানোর একটি ভাল উপায়৷ একটি উৎপাদন পরিবেশে, আপনাকে সেই কন্টেইনারগুলিকে পরিচালনা করতে হবে যা অ্যাপ্লিকেশনগুলি চালায় এবং নিশ্চিত করতে হবে যে কোনও ডাউনটাইম নেই৷ উদাহরণস্বরূপ, যদি একটি ধারক ডাউন হয়ে যায়, তখন অন্য ধারকের শুরু হতে হবে। এই আচরণ একটি সিস্টেম দ্বারা পরিচালিত হলে এটি সহজ হবে না?
+
+এভাবেই কুবারনেটিস উদ্ধারে আসে! কুবারনেটিস আপনাকে সিস্টেমগুলিকে স্থিতিস্থাপকভাবে চালানোর জন্য একটি কাঠামো প্রদান করে। এটি আপনার অ্যাপ্লিকেশনের জন্য স্কেলিং এবং ফেইলওভারের যত্ন নেয়, স্থাপনার নিদর্শন প্রদান করে এবং আরও অনেক কিছু করে। উদাহরণস্বরূপ, কুবারনেটিস সহজেই আপনার সিস্টেমের জন্য একটি ক্যানারি স্থাপনা পরিচালনা করতে পারে।
+
+কুবারনেটিস আপনাকে সরবরাহ করে:
+
+* **পরিষেবা আবিষ্কার এবং লোড ব্যালেন্সিং**
+কুবারনেটিস ডিএনএস নাম ব্যবহার করে বা তাদের নিজস্ব আইপি ঠিকানা ব্যবহার করে একটি ধারক প্রকাশ করতে পারে। একটি কন্টেইনারে ট্রাফিক বেশি হলে, কুবারনেটিস লোড ব্যালেন্স এবং নেটওয়ার্ক ট্র্যাফিক বিতরণ করতে সক্ষম হয় যাতে স্থাপনা স্থিতিশীল থাকে।
+* **স্টোরেজ অর্কেস্ট্রেশন**
+কুবারনেটিস আপনাকে স্বয়ংক্রিয়ভাবে আপনার পছন্দের একটি স্টোরেজ সিস্টেম মাউন্ট করার অনুমতি দেয়, যেমন স্থানীয় স্টোরেজ, পাবলিক ক্লাউড প্রদানকারী এবং আরও অনেক কিছু।
+* **স্বয়ংক্রিয় রোলআউট এবং রোলব্যাক**
+আপনি কুবারনেটিস ব্যবহার করে আপনার স্থাপন করা কন্টেইনার জন্য পছন্দসই অবস্থা বর্ণনা করতে পারেন এবং এটি একটি নিয়ন্ত্রিত হারে প্রকৃত অবস্থাকে পছন্দসই অবস্থায় পরিবর্তন করতে পারে। উদাহরণস্বরূপ, আপনি আপনার স্থাপনার জন্য নতুন কন্টেইনার তৈরি করতে, বিদ্যমান কন্টেইনারগুলি সরাতে এবং নতুন কন্টেইনারে তাদের সমস্ত রিসোরস গ্রহণ করতে কুবারনেটসকে স্বয়ংক্রিয়ভাবে করতে পারেন।
```

Comment:
Kindly check this sentence `কুবারনেটসকে স্বয়ংক্রিয়ভাবে করতে পারেন।` . Specially the word `রিসোরস` & `কুবারনেটসকে`